src/MAUI/Maui.Samples/Views/CategoryPage.xaml.cs

```diff
@@ -44,6 +48,31 @@ private void Initialize()
 
     }
 
+    private async void FeedbackToolbarItem_Clicked(object sender, EventArgs e)
+    {
+        await DisplayActionSheet("Open an issue on GitHub:", "Cancel", null, [BugReport, FeatureRequest]).ContinueWith((result) =>
+        {
+            string link;
+            switch (result.Result)
+            {
+                case BugReport:
```

Comment:
Looks like we aren't including the impacted samples in the link on MAUI.

Reply:
This issue is specific to the .NET MAUI implementation of feedback functionality. Current.SelectedSample is [set to null when navigated from sample page](https://github.com/Esri/arcgis-maps-sdk-dotnet-samples/blob/cad67fb33713c15024b55514f52a36c0e56c56f5/src/MAUI/Maui.Samples/Views/SamplePage.xaml.cs#L131). This toolbar item I added to the Category page and not the Sample page for a few reasons:
1. to not overcrowd the SamplePage toolbar
1. to match the WinUI and WPF implementations which place the feedback button next to the settings button
1. a feature request for the sample viewer isn't likely specific to a single sample

That being said, I've added a `PreviouslySelectedSample` property for bug reporting purposes which works as a simple fix. In this scenario, a user would find a bug in the sample, navigate back to a category page, click the feedback button, open a bug report with the previously selected sample autofilled.

Reply:
I think this is a partial solution but I don't think it fully addresses the issue. 

A user may not want the previously selected sample to be automatically populated. 

If it's not possible to add the bug/feature feedback option when the sample is open I think we leave it unpopulated and let the user enter the appropriate data.

Reply:
It is technically possible, so I undid this change and added the feedback button to the sample page toolbar per discussion with @pMaske